lib/interpret_and_optimize/services/pb_visual_generation_service.dart

```diff
@@ -95,7 +95,7 @@ class PBVisualGenerationService implements PBGenerationService {
               smHelper.interpretStateManagementNode(result);
             } else {
               smHelper.interpretStateManagementNode(result);
-              continue;
+              //continue;
```

Comment:
The lack of this `continue` causes parabeac-core to contain duplicate files, since the states are not being removed from the tree. Here's an example:

![Screen Shot 2021-05-24 at 11 02 05 AM](https://user-images.githubusercontent.com/42812006/119382685-9a44ca00-bc7f-11eb-821a-c67f86d66cfb.png)

Reply:
Ah, yes.  But this is a conversation we had about removing that continue because states get removed too early I believe, I'll check it out.

Reply:
I think a possible solution we discussed was adding a boolean to the intermediate nodes to prevent them from generating once we know they're states. I'll try that as well 🤔

Reply:
Fixed in new code will try and edit pull request